src/integration/createMetaInfo.js

```diff
@@ -39,6 +40,7 @@ function latestRepoCloner(config) {
     const branch = config.generationResult.branch;
     const expectedCloneLocation = config.generationResult.clonedResultFolderPath;
 
+    logger.info(`Cloning main repository...`);
```

Comment:
Main repository? Can we call it somehow different? For example in config we have generationResult.cloneLocation and it will give us the name latestResultRepo ?